client/components/App.jsx

```diff
@@ -0,0 +1,39 @@
+// This component handles the App template used on every page.
```

Comment:
Shouldn't this comment be a documentation comment on above class declaration line ?